embed_grader/serapis/forms/course_forms.py

```diff
@@ -103,6 +103,7 @@ def save_and_commit(self):
             assign_perm('view_assignment', student_group, course)
             assign_perm('modify_assignment', instructor_group, course)
             assign_perm('create_assignment', instructor_group, course)
+            assign_perm('download_csv_data', instructor_group, course)
```

Comment:
Now look back our permission mechanism is not well designed... It's over complicated now. Let's not declare a new permission because the existing users will not get this one. Instead, let's use `create_assignment` or `modify_course`